configs/coding_help.json

```
@@ -1,8 +1,7 @@
 {
   "index_name": "coding_help",
   "start_urls": [
-    "https://help.coding.net/docs/",
-    "https://help.coding.net/docs/start/create-enterprise.html"
+    "https://help.coding.net/docs/start/new.html"
```

```suggestion
    "https://help.coding.net/docs/start/new.html",
    "https://help.coding.net/docs/"
```

The base URL should also be stated to avoid only scraping `new.html`